cmd/batcher/main.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2019 kubeflow.org.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"os"
+	"strconv"
+	"github.com/kubeflow/kfserving/pkg/batcher"
+	"github.com/kubeflow/kfserving/pkg/batcher/controllers"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+var (
+	port          = flag.String("port", "8082", "Logger port")
```

Comment:
Should be Batcher port rather than "Logger port" right?

Reply:
I will modify it.